app/src/main/java/com/keylesspalace/tusky/interfaces/AnchorActivity.kt

```diff
@@ -0,0 +1,7 @@
+package com.keylesspalace.tusky.interfaces
+
+import android.view.View
+
+interface AnchorActivity {
```

Comment:
It worked not correct because activity has a scrollable toolbar and fragments have CoordinatorLayout as root. When app searches the layout for snackbar it founds the fragment's CoordinatorLayout. But root CoordinatorLayout hides the bottom of the fragment (with Snackbar) when toolbar shown. I've removed child CoordinatorLayout, so, Snackbar always uses the Activity's CoordinatorLayout